docs/mp/config/01_introduction.adoc

```diff
@@ -41,13 +41,11 @@ include::{common-deps-page-prefix-inc}[tag=maven-dependency]
         </dependency>
 ----
 
-== About {spec-name}
+== Overview
```

Comment:
Why is this changed? I thought this was used in other docs as well.

Reply:
It different across chapters e.g. sometimes just spec-name, sometimes overview, sometimes introductions etc. For this, I reverted to old way.